src/components/bodhi-dls/cardFinished.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component, PropTypes } from 'react';
+import { Button, Alert } from 'antd';
+import { connect } from 'react-redux';
+
+import topicActions from '../../redux/topic/actions';
+
+class CardFinished extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+    };
+
+    this.onWithdrawBtnClicked = this.onWithdrawBtnClicked.bind(this);
+  }
+
+  componentWillMount() {
+  }
+
+  onWithdrawBtnClicked() {
+    // Withdraw action
+
+    this.props.onSubmit();
+  }
+
+  render() {
+    const {
+      amount, config, children, result,
+    } = this.props;
+
+    const amountStr = amount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+
+    // Build Alert elements based on result
+    let alertElement;
+
+    if (result) {
```

Comment:
is  `result` ever true here? don't see any logic to set `result` before this?

Reply:
Yeah, my bad. Made an update to mark it required and passed down from topic.js